dev-requirements.txt

```diff
@@ -1,4 +1,5 @@
 black
+build
```

Comment:
What exactly in the other files refer to this package, I don't see how it gets engaged?

Reply:
In principle, `build` is required to build the project: https://packaging.python.org/en/latest/tutorials/packaging-projects/#generating-distribution-archives

But I see that `pip install .` seems to work either way. I generally build dists (wheel and sdist) with `python -m build` but I guess that's not strictly necessary if you're not uploading them anywhere.

Will do some more testing.

Reply:
Have tested everything without `build` in my env and indeed it's all good, you just can't build distributions, which is fine. Removed it from the requirements.